boot-chaos-monkey/README.md

```
@@ -1,14 +1,23 @@
-# spring-boot-chaos-monkey
+# Spring Boot Chaos Monkey
 
 [Choas Monkey](https://netflix.github.io/chaosmonkey/)
 
+This module explores Chaos Engineering principles by intentionally injecting failures into a Spring Boot application to test its resilience. It demonstrates how to set up and integrate `chaos-monkey-spring-boot` for your microservices, as well as performance testing with Gatling to measure any degradation under chaos scenarios.
+
 
 ![](../images/chaos-monkey.png)
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)